js/index.js

```diff
@@ -42,8 +42,40 @@ window.addEventListener('load', event => {
   // Bind the click event of each element to a function
   document.querySelectorAll('.card').forEach(card => {
     card.addEventListener('click', () => {
-      // TODO: write some code here
-      console.log(`Card clicked: ${card}`);
+      let cardName = card.getAttribute("data-card-name");
+      console.log(`card picked is ${cardName}`);
+      card.querySelectorAll('.card div').forEach(div =>{
+        div.classList.toggle("front")
+        div.classList.toggle("back")
+      })
+      console.log(`bla`,memoryGame.pickedCards)
+      memoryGame.pickedCards.push(cardName)
+      console.log(`nlanla`,memoryGame.pickedCards)
+      let card1 = memoryGame.pickedCards[0];
+      let card2 = memoryGame.pickedCards[1];
+      //console.log(memoryGame.checkIfPair(memoryGame.pickedCards[0],memoryGame.pickedCards[1]))
+      if(memoryGame.pickedCards.length === 2){
+      if (memoryGame.checkIfPair(card1, card2) === true){
+        console.log(`it is a match !`)
+        let pairsGuessed = document.getElementById("pairs-guessed")
+        pairsGuessed.innerHTML++;
+        let pairsClicked = document.getElementById("pairs-clicked") ;
+        pairsClicked.innerHTML++;
+        memoryGame.pickedCards.splice(0, memoryGame.pickedCards.length);
+
+        //memoryGame.pickedCards =[];
+      }else if (memoryGame.checkIfPair(card1, card2) === false){
+        let pairsClicked = document.getElementById("pairs-clicked") ;
+        pairsClicked.innerHTML++;
+        memoryGame.pickedCards.splice(0, memoryGame.pickedCards.length);
+        console.log(memoryGame.pickedCards);
+
```

Comment:
The game is more realistic if you get the faced up cards to flip back down if they're not a pair.

That's why you should use `memoryGame.pickedCards.push(card)` on line 52 because then `card1` and `card2` are both points of reference and so you can use them to dynamically flip down the cards.
```
setTimeout(() => {
          card1.querySelectorAll('.card div').forEach(div =>{
            div.classList.toggle("front")
            div.classList.toggle("back")
          })
  
          card2.querySelectorAll('.card div').forEach(div =>{
            div.classList.toggle("front")
            div.classList.toggle("back")
          })
        },500)